infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```diff
@@ -235,6 +235,30 @@ param parTags object = {}
 @sys.description('Set Parameter to true to Opt-out of deployment telemetry.')
 param parTelemetryOptOut bool = false
 
+@sys.description('Define inbound destination port or a range for HTTPS. Default:443')
+param parHttpsPort string = '443'
+
+@sys.description('Define inbound destination port or a range for gateway manager. Default:443')
+param parGatewayManagerPort string  ='443'
+
+@sys.description('Define inbound destination port or a range for load balancer. Default:443')
+param parLoadBalancerPort string = '443'
+
+@sys.description('Define inbound destination ports or ranges for Bastion Host Communication. Default:[8080, 5701]')
+param parBastionHostCommunicationPorts array = ['8080','5701']
+
+@sys.description('Define outbound destination ports or ranges for SshRDPOutbound. Default:[22,3389]')
+param parSshRDPPort array = ['22','3389']
+
+@sys.description('Define outbound azure cloud destination port range for Azure cloud. Default:443')
+param parAzureCloudPort string = '443'
+
+@sys.description('Define outbound destination port ranges for Bastion Communication. Default:[8080, 5701]')
+param parBastionCommunicationPorts array = ['8080','5701']
+
+@sys.description('Define outbound get session destination port range for Get Session Information. Default:80')
+param parGetSessionPort string = '80'
+
```

Comment:
Can we make these names or descriptions a little more descriptive around what they are used fro. This module does an awful lot more than just deploy an NSG, so having something like `parHttpsPort` is to generic for this module.

For example `parHttpsPort` could/should be something like `parBastionInboundHttpsPort`

Please review and amend all of theses new parameters with this review comment in mind,

Reply:
Thanks for the review @jtracey93. It totally makes sense to only parameterize the SSH/RDP port only and not others. I have updated the PR and docs for it.